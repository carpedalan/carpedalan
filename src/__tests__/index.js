import React from 'react';
import { create } from 'react-test-renderer';

import Root from '..';

jest.mock('superagent', () =>
  jest.fn().mockReturnValue({ body: { data: 'data' } }),
);

describe('test', () => {
  it('should do some stuff', () => {
    const instance = create(<Root />);
    expect(instance.toJSON()).toMatchSnapshot();
  });
});
