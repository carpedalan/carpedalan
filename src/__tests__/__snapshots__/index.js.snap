// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test should do some stuff 1`] = `
Array [
  <div
    onClick={[Function]}
  >
     click me
  </div>,
  <a
    href="/login"
    onClick={[Function]}
  >
    asdfwefwefwef
  </a>,
  <div />,
  "no match",
]
`;
